results.csv

```diff
@@ -1,6 +1,13 @@
-1,true
```

Comment:
Finde die Datei `results.csv` sollten wir zur `.gitignore` hinzufügen. Diese Datei also nicht in den Code aufnehmen - sie entsteht ja erst beim Ausführen.